serverless/packages/functions/src/datasets/user/delete_userdata.ts

```diff
@@ -0,0 +1,34 @@
+import { APIGatewayProxyHandlerV2 } from "aws-lambda";
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
+import { S3Client, DeleteObjectCommand } from "@aws-sdk/client-s3";
+import parseJwt from "@dlp-sst-app/core/parseJwt";
+
+export const handler: APIGatewayProxyHandlerV2 = async (event) => {
+    if (
+        event &&
+        event.pathParameters &&

```

Comment:
why is this file called delete_userdata.ts. I'm confused on what the code here has to do with user data @alantao912